schemes/musica/musica_ccpp.F90

```diff
@@ -23,28 +23,40 @@ subroutine musica_ccpp_register(solver_type, num_grid_cells, constituent_props,
     character(len=512),                               intent(out) :: errmsg
     integer,                                          intent(out) :: errcode
 
-    call micm_register(solver_type, num_grid_cells, constituent_props, errmsg, errcode)
+    type(ccpp_constituent_properties_t), allocatable :: constituent_set(:)
+
+    call micm_register(solver_type, num_grid_cells, constituent_set, errmsg, errcode)
+    if (errcode /= 0) return
+    constituent_props = constituent_set
+
+    call tuvx_register(constituent_set, errmsg, errcode)
+    if (errcode /= 0) return
+    constituent_props = [ constituent_props, constituent_set ]
```

Comment:
Another thought is that these two variable names `constituent_props` and `constituent_set` don't seem informative. What does 'set' mean differently from 'props'?

Reply:
Good point. These weren't good name choices. Maybe `constituent_props_subset` instead? to indicate that it is just used to temporarily hold the specific species needed for MICM, and later TUV-x?

Reply:
Yeah, that sounds good to me. Thank you!